jaxrs/src/main/java/org/killbill/billing/jaxrs/util/JaxrsUriBuilder.java

```diff
@@ -74,6 +79,12 @@ public URI getUri() {
         return ri.entity(obj).build();
     }
 
+    private UriBuilder getUriBuilder(final String path, final Class<? extends JaxrsResource> theClassMaybeEnhanced, @Nullable final String getMethodName) {
+        final Class theClass = getNonEnhancedClass(theClassMaybeEnhanced);
+        return getMethodName != null ? UriBuilder.fromPath(path.equals("/") ? path.substring(1) : path).path(theClass).path(theClass, getMethodName) :
+               UriBuilder.fromPath(path).path(theClass);
+    }
+
     private UriBuilder getUriBuilder(final Class<? extends JaxrsResource> theClassMaybeEnhanced, @Nullable final String getMethodName) {
```

Comment:
@sbrossie yes, it is still in use by 2 methods [here](https://github.com/javier-gomez-hs/killbill/blob/jgomez-non-root-issues/jaxrs/src/main/java/org/killbill/billing/jaxrs/util/JaxrsUriBuilder.java#L70) and [here](https://github.com/javier-gomez-hs/killbill/blob/jgomez-non-root-issues/jaxrs/src/main/java/org/killbill/billing/jaxrs/util/JaxrsUriBuilder.java#L57).
A refactor might be needed, but will need to have the "path" value.